src/main/java/org/ibp/api/java/impl/middleware/inventory/manager/LotServiceImpl.java

```diff
@@ -198,9 +192,24 @@ public LotSearchMetadata getLotsSearchMetadata(final LotsSearchDto lotsSearchDto
 	@Override
 	public void closeLots(final LotsSearchDto searchDTO) {
 		final List<ExtendedLotDto> lotDtos = this.lotService.searchLots(searchDTO, null);
-		extendedLotListValidator.validateClosedLots(lotDtos.stream().collect(Collectors.toList()));
+		extendedLotListValidator.validateClosedLots(lotDtos);
 		final WorkbenchUser loggedInUser = this.securityService.getCurrentlyLoggedInUser();
 		lotService.closeLots(loggedInUser.getUserid(), lotDtos.stream().map(ExtendedLotDto::getLotId).collect(Collectors.toList()));
 	}
 
+	@Override
+	public void mergeLots(final String keepLotUUID, final LotsSearchDto lotsSearchDto) {
+		final List<ExtendedLotDto> lotDtos = this.lotService.searchLots(lotsSearchDto, null);
+		this.lotMergeValidator.validate(keepLotUUID, lotDtos);
+		lotsSearchDto.setLotIds(lotDtos.stream().map(LotDto::getLotId).collect(Collectors.toList()));
```

Comment:
seems like this assignment is not needed @lik-83

Reply:
Done!